recipes-omnect/images/omnect-os-image.bb

```diff
@@ -118,3 +117,60 @@ python () {
 }
 
 inherit omnect_user
+
+inherit logging
+
+# postive test for packages, i.e. check if installed
+check_installed_packages() {
+    local manifest="$1"
+    local pkglist="$2"
+    local error_not_installed="$3"
+    local ret=0
+    for p in $pkglist; do
+        if ! grep -q "^$p " "$manifest"; then
+            [ "$error_not_installed" = 1 ] \
+               && bberror "Required TOOL not installed in image: $p"
+            ret=1
+         fi
+    done
+    return $ret
+}
+
+# post processing function checking for default tools
+verify_image_tools() {
+    local ret=0
+    local manifest="${IMAGE_MANIFEST}"
+    local release="${OMNECT_RELEASE_IMAGE}"
+    local tools="${OMNECT_DEVEL_TOOLS}"
+    local relhint=""
+
+    # something to be checked at all?
+    [ "$tools" ] || return 0
+
+    [ "$release" = 1 ] && relhint="release "
+    bbnote "NOTE: checking for OMNECT_DEVEL_TOOLS in ${relhint}image ..."
+    bbnote "[$tools]"
+
+    set -- ${tools}
+    if [ $# -gt 0 ]; then
+        if [ "$release" = 1 ]; then
+            if check_installed_packages "$manifest" "$tools" 0; then
+                bbwarn 'OMNECT_DEVEL_TOOLS are missing in image!'
```

Comment:
this is the `release = 1` case ... so that should be the case

Reply:
uhh, messages were swapped ...